sbi/neural_nets/density_estimators/base.py

```diff
@@ -48,11 +72,66 @@ def loss(self, input: Tensor, condition: Tensor, **kwargs) -> Tensor:
     def sample(self, sample_shape: torch.Size, condition: Tensor, **kwargs) -> Tensor:
         r"""Return samples from the density estimator.
 
+        Note:
+            This function should support batched conditions and should admit the
+            following behavior for different condition shapes:
+            - (*condition_shape) -> (*sample_shape,d)
```

Comment:
what is `d` here? is it the number of dimensions of the PDF itself?

Reply:
Yes, I use d consistently. But I can also give it a more descriptive name i.e. "input_dims"

Reply:
both is fine if we mention it somewhere explicitly 👍